bulk_deploy/deploy_actions.rb

```diff
@@ -0,0 +1,396 @@
+          postgres: flask_service_name,
+          git: true,
+          units: units_per_app
+        )
+
+        # ################### Deploy Digital Marketplace Search API backend ###################
+        # dm_search_api_app_name = "dm-search-api-app-" + Time.now.to_i.to_s.reverse
+        # dm_search_api_service_name = "es-" + Time.now.to_i.to_s.reverse
+        # log_dict[team][users_in_team[2][:email]] = {
+        #   :app => dm_api_app_name,
+        #   :service => dm_api_service_name
+        # }
+        # deploy_client.deploy_app(
+        #   user: users_in_team[2],
+        #   app: {
+        #     name: dm_api_app_name,
+        #     dir: dm_api_app_dir,
+        #     platform: "python"
+        #   },
+        #   env_vars: {
+        #     DM_SEARCH_API_AUTH_TOKENS: "oursearchtoken",
+        #   },
+        #   units: units_per_app
+        # )
+        #
+        # ################### Deploy Digital Marketplace API backend ###################
+        # dm_api_app_name = "dm-api-app-" + Time.now.to_i.to_s.reverse
+        # dm_api_service_name = "db-" + Time.now.to_i.to_s.reverse
+        # log_dict[team][users_in_team[3][:email]] = {
+        #   :app => dm_api_app_name,
+        #   :service => dm_api_service_name
+        # }
+        # deploy_client.deploy_app(
+        #   user: users_in_team[3],
+        #   app: {
+        #     name: dm_api_app_name,
+        #     dir: dm_api_app_dir,
+        #     platform: "python"
+        #   },
+        #   env_vars: {
+        #     DM_API_AUTH_TOKENS: "ourtoken",
+        #     DM_SEARCH_API_AUTH_TOKEN: "CHbDLQtMvKoAuAtT8GM6vrdGGC",
+        #     DM_SEARCH_API_URL: "https://preview-search-api.development.digitalmarketplace.service.gov.uk"
+        #   },
+        #   units: units_per_app
+        # )
+
+        ################### Deploy Digital Marketplace Supplier frontend app ###################
+        dm_supplier_frontend_app_name = "dm-supplier-frontend-app-" \
+        + Time.now.to_i.to_s.reverse
+        log_dict[team][users_in_team[4][:email]] = {
+          :app => dm_supplier_frontend_app_name
+        }
+        deploy_client.deploy_app(
+          user: users_in_team[4],
+          app: {
+            name: dm_supplier_frontend_app_name,
+            dir: dm_supplier_frontend_app_dir,
+            platform: "python"
+          },
+          env_vars: {
+            DM_ADMIN_FRONTEND_COOKIE_SECRET: "secret",
+            DM_ADMIN_FRONTEND_PASSWORD_HASH: "JHA1azIkMjcxMCRiNWZmMjhmMmExYTM0OGMyYTY0MjA3ZWFkOTIwNGM3NiQ4OGRLTHBUTWJQUE95UEVvSmg3djZYY2tWQ3lpcTZtaw==",
+            DM_DATA_API_AUTH_TOKEN: "wXeLg9vQNRqdkb9kccHDzFRaNL",
+            DM_DATA_API_URL: "https://preview-api.development.digitalmarketplace.service.gov.uk",
+            DM_MANDRILL_API_KEY: "somekey",
+            DM_PASSWORD_SECRET_KEY: "verySecretKey",
+            DM_S3_DOCUMENT_BUCKET: "admin-frontend-dev-documents",
+            DM_SEARCH_API_AUTH_TOKEN: "CHbDLQtMvKoAuAtT8GM6vrdGGC",
```

Comment:
These are tokens to the preview environments of Digital Marketplace. We will be replacing these soon with our generic tokens (like `ourtoken` and `oursearchtoken`) soon as we are deploying our own instances of both backend apps (API and Search API apps) therefor we won't rely on the preview environment anymore.

The next PR after this one will replace the Postgres API app with our own and the PR after that will replace the Search API app with our own so these tokens will be gone in next two PRs.